app/jobs/pageflow/reset_counter_caches.rb

```diff
@@ -0,0 +1,13 @@
+module Pageflow
+  class ResetCounterCachesJob
```

Comment:
This should be a migration instead. Using ActiveRecord models from migrations is not allowed though (the (future) schema assumed by the models might not match the schema at that point during the migration process), so we'll have to go with raw SQL queries here ([see example](https://github.com/codevise/pageflow/blob/master/db/migrate/20151022080518_add_first_published_at_to_entries.rb#L5))